packages/client/src/certificate/mutations.ts

```diff
@@ -37,14 +43,14 @@ async function updateCertificateTemplate(
     status: status,
     event: event
   }
-  return (
+  const response =
     client &&
-    client.mutate({
+    (await client.mutate({
       mutation: UPDATE_CERTIFICATE_TEMPLATE,
-      variables: { certificateSVG },
-      refetchQueries
-    })
-  )
+      variables: { certificateSVG }
+    }))
+
+  return response.data
```

Comment:
In the case of `client` being undefined, this could throw an error. Could you see what's wrong with `client` not being typed properly?

Reply:
Fixed the type. It was defined as any instead of the actual type.